aws/eks-cluster/iam.tf

```diff
@@ -177,7 +177,7 @@ data "aws_iam_policy_document" "node_assume_role_policy" {
 
     principals {
       type        = "Service"
-      identifiers = ["eks.amazonaws.com"]
+      identifiers = ["ec2.amazonaws.com"]
```

Comment:
Why do we change to ec2?

Reply:
>Change assume policy: to match with bridgehealth-ai eks cluster, and document https://docs.aws.amazon.com/eks/latest/userguide/create-node-role.html. I don't understand why the original assume policy was eks.amazonaws.com for the node role though.

It was ec2 in bridgehealth-ai terraform module https://github.com/bridgehealth-ai/infra-terraform/blob/master/modules/aws/eks-cluster/iam.tf#L175